plugins/web/opentelemetry-instrumentation-document-load/src/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
You need to export this file from `index.ts`, otherwise the interfaces it provides are not available for users to consume.

```js
// index.ts

export * from './types';
```